apps/web/src/pages/settings/organization/OrganizationPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { PageContainer } from '@novu/design-system';
+import { useAuthContext } from '@novu/shared-web';
+import PageHeader from '../../../components/layout/components/PageHeader';
+import { css } from '../../../styled-system/css';
+import { HStack } from '../../../styled-system/jsx';
+import { OrganizationLogo } from './OrganizationLogo';
+import { OrganizationName } from './OrganizationName';
+
+const PAGE_TITLE = 'Organization profile';
+
+export function OrganizationPage() {
+  const { currentOrganization } = useAuthContext();
+
+  return (
+    <PageContainer title={PAGE_TITLE}>
+      <PageHeader title={PAGE_TITLE} />
+
+      <div className={css({ p: 24 })}>
+        <HStack gap={24} alignItems="flex-end" h={72}>
```

Comment:
```suggestion
        <Stack gap={'150'} direction="row">
```
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Ideally should let content dictate the size here.

<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** `HStack` and `VStack`‏‏ are opinionated in that the center the items along their main axis

Reply:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice use of a Panda Pattern!‏

Reply:
So, `HStack` is basically a flex container with `justify-content: center` ?
and we should use normal `Stack` for most scenarios?

Reply:
Correct, VStack and HStack are fairly similar to Stack, but slightly more opinionated: https://panda-css.com/docs/concepts/patterns#hstack